functional/FunctionalTest.java

```diff
@@ -515,7 +515,7 @@ public static void putObject_test6() throws Exception {
 
     long startTime = System.currentTimeMillis();
     try {
-      String objectName = "/path/to/" + getRandomName();
```

Comment:
'/path/to/` is also a valid input according to S3 specification. You would need to add a new test case.

Reply:
It is not valid for MinIO @balamurugana also we never preserved the "/" in the beginning. We happily trimmed all "/". So we shall keep the test case that works with MinIO as expected.

Reply:
What I understand is, minio server does not support `/path/to/`. If true, my comment can be ignored.

Reply:
Your understanding is correct.